met-web/src/components/engagement/view/widgets/PhasesWidget/PhasesWidgetMobile/EngagementPhaseMobile.tsx

```diff
@@ -12,32 +14,42 @@ export const EngagementPhaseMobile = ({
     learnMoreText,
     popOverText,
     accordionBackground,
+    phaseId,
 }: ProcessStageProps) => {
+    const { widgets } = useContext(ActionContext);
+    const phasesWidget = widgets.find((widget: Widget) => widget.widget_type_id === WidgetType.Phases);
+    const currentPhase = phasesWidget?.items[0]?.widget_data_id || EngagementPhases.Standalone;
+    const isCurrent = phaseId >= currentPhase;
+
     return (
         <PhaseBoxMobile
             title={title}
-            backgroundColor={backgroundColor}
+            backgroundColor={isCurrent ? backgroundColor : PAST_PHASE.backgroundColor}
             learnMoreBox={
-                <ReadMoreBox backgroundColor={accordionBackground} sx={{ p: 0, margin: 0 }}>
+                <ReadMoreBox
+                    backgroundColor={isCurrent ? accordionBackground : PAST_PHASE.backgroundColor}
+                    sx={{ p: 0, margin: 0 }}
+                >
                     <Grid
-                        sx={{ color: 'black' }}
+                        sx={{ color: isCurrent ? 'black' : 'white' }}
                         container
                         direction="row"
                         justifyContent="flex-start"
                         alignItems="flex-start"
                         spacing={1}
                     >
                         <Grid item xs={12}>
-                            <MetParagraph sx={{ fontWeight: 'bold', fontSize: '1.3rem', color: 'black' }}>
-                                {title}
-                            </MetParagraph>
+                            <MetParagraph sx={{ fontWeight: 'bold', fontSize: '1.3rem' }}>{title}</MetParagraph>
```

Comment:
weird that we are setting fontSize. This should be a standard by now. We have created MetHeaders and MetParagraph so we don't have to worry about setting fontSize anymore. What made us do it here?

Reply:
I think i was getting a unit test error saying a header tag cannot be within a div tag

Reply:
I reverted it to MetHeader4 there was a jest bug before but I think it got patched.